tests/e2e/package.json

```diff
@@ -37,6 +37,8 @@
     "edit-json-file": "1.7.0",
     "eslint": "7.32.0",
     "eslint-plugin-import": "2.24.2",
+    "redis": "^3.0.2",
+    "redis-rejson": "^1.0.0",
```

Comment:
With node-redis, you have bundled all modules clients

Reply:
yes, I see, thanks
deleted redis-rejson